lang/test/org/partiql/lang/errors/ParserErrorsTest.kt

```diff
@@ -1246,6 +1246,96 @@ class ParserErrorsTest : TestBase() {
             Property.TOKEN_TYPE to TokenType.EOF,
             Property.TOKEN_VALUE to ion.newSymbol("EOF")))
 
+    @Test
+    fun updateWithNestedSet() = checkInputThrowingParserException(
+        "UPDATE test SET x = SET test.y = 6",
```

Comment:
So when I'm testing this query in the REPL, it gives the following:

```
PartiQL> UPDATE test SET x = SET test.y = 6;
org.partiql.lang.syntax.ParserException: Type UPDATE only expected at the top level
	Parser Error: at line 1, column 21: unexpected term found, KEYWORD : set
```
Shouldn't the 2nd line say `SET` and not `UPDATE`? I haven't had a chance to test w/ the other top-level tokens.

Reply:
The associated parse type is `UPDATE`.
I will change this to - `"Keyword ${node.token} only expected at the top level"`.
It makes more sense I guess and would be less confusing.